src/components/Campaign/CampaignPage.tsx

```diff
@@ -1,35 +1,25 @@
-'use client';
-
-import type { CampaignData } from '@/types/strapi';
-import { useTranslation } from 'react-i18next';
-import { JUMPER_PROFILE_PATH } from 'src/const/urls';
 import { QuestDataExtended } from 'src/types/merkl';
-import { MerklRewards } from '../ProfilePage/MerklRewards';
-import { PageContainer } from '../ProfilePage/ProfilePage.style';
-import { QuestsOverview } from '../ProfilePage/QuestsOverview/QuestsOverview';
-import { BackButton } from '../QuestPage/BackButton/BackButton';
-import { CampaignHeader } from './CampaignHeader/CampaignHeader';
+import { CampaignData } from 'src/types/strapi';
+import { CampaignHero } from './CampaignHero/CampaignHero';
+import { MissionsSection } from './MissionsSection/MissionsSection';
+import { MissionsList } from './MissionsSection/MissionsList';
+import { GridContainer } from '../Containers/GridContainer';
+import { PageContainer } from '../Containers/PageContainer';
 
 interface CampaignPageProps {
   campaign: CampaignData;
   quests: QuestDataExtended[];
 }
 
 export const CampaignPage = ({ campaign, quests }: CampaignPageProps) => {
-  const { t } = useTranslation();
   return (
-    <PageContainer className="profile-page">
-      <BackButton
-        path={JUMPER_PROFILE_PATH}
-        title={t('navbar.navbarMenu.profile') || 'Profile'}
-      />
-      {campaign.merkl_rewards && (
-        <MerklRewards merklRewards={campaign.merkl_rewards} />
-      )}
-      <CampaignHeader campaign={campaign} />
-      {Array.isArray(campaign.quests) && campaign.quests?.length > 0 && (
-        <QuestsOverview quests={quests} label={campaign.Slug} />
-      )}
+    <PageContainer>
+      <CampaignHero campaign={campaign} />
+      <MissionsSection>
+        <GridContainer>
+          <MissionsList missions={quests} />
+        </GridContainer>
+      </MissionsSection>
```

Comment:
Maybe we could avoid the empty section here by checking beforehand. Or possibly have an alternative solution for this we can discuss with design.
<img width="2736" height="610" alt="image" src="https://github.com/user-attachments/assets/51ae5119-8bc2-4388-9a8b-854b56ea7970" />

```suggestion
      {Array.isArray(quests) && quests.length > 0 && (
        <MissionsSection>
          <GridContainer>
            <MissionsList missions={quests} />
          </GridContainer>
        </MissionsSection>
      )}
```